### DISCUSSION TITLE
How to inflate cortical mesh?

### DISCUSSION THREAD

**Discussion by AnnaBhlr** [brainvisa/brainvisa.github.io#156] [Q&A] [answered]
Hi,
I have the cortical and white matter mesh for each hemisphere from T1w images as a result of the morphologist pipeline (as Gifti). I'd like to inflate each hemisphere using Brainvisa. I read about that brainvisa/toolboxes/morphologist/processes/surface/inflateCorticalSurface.py in the Docs but not sure if it runs with my meshes as .gii and how I would call it via the command line then.
Merci, Anna

**Comment by denisri** [comment, marked as answer]
Hi,
Yes the `InflateCorticalSurface` process is what you need, and yes, sure, it can read and write .gii format meshes.
You can run any BrainVisa process or pipeline via a commandline, using the commandline `axon-runprocess` (or the equivalent, `python -m brainvisa.axon.runprocess`. Run it using the parameter `-h` to get the help. Here you would use something like:
```
axon-runprocess InflateCorticalSurface input_mesh=mesh.gii output_mesh=inflated_mesh.gii curvature_texture=mesh_curv.gii
```
(you need to replace files names with yours, of course)
If using a BrainVisa database and if you want to use filenames completion (output deduced from the input), you need to use the option `--enabledb`.
You can also iterate the process over a set of meshes, using the option `-i`. Using a brainvisa database, it could look like this:
```
axon-runprocess --enabledb -i input_mesh InflateCorticalSurface input_mesh="['/data/bv_base/subjects/s1/t1mri/default_acquisition/default_analysis/segmentation/mesh/s1_Lhemi.gii', '/data/bv_base/subjects/s1/t1mri/default_acquisition/default_analysis/segmentation/mesh/s1_Rhemi.gii']"
```

Denis

**Reply by AnnaBhlr** [reply]
Thanks! That helped a lot!
